.github/workflows/full_suite_integration_tests.yml

```diff
@@ -99,6 +99,8 @@ jobs:
 
       - name: Prepare Google Service Account
         env:
+          FLANK_PROJECT_ID: flank-open-source
+          GOOGLE_CLOUD_PROJECT: flank-open-source
```

Comment:
Hey @Sloox I think that is a bad workaround for the real problem, project id should be mocked for testing

Reply:
@piotradamczyk5 do we have an example for this? I have looked and my windows machines looks for it this way.

Reply:
its also not needed on ubuntu and mac so its confused me a bit.

Reply:
If we need to have it set, then make it happen in integration tests Gradle, but I am pretty sure that it could be taken from the service account file. It works that way previously

Reply:
@Sloox 
Check this doc please: https://flank.github.io/flank/#authenticate-with-a-service-account

Reply:
@piotradamczyk5 removed. @adamfilipow92